packages/@lwc/wds-playground/src/browser/ui/pg/playground/playground.js

```diff
@@ -47,6 +50,56 @@ export default class Playground extends LightningElement {
     });
   }
 
+  isDOMRectWithinBound(componentDOMRect) {
+    const containerElement = this.template.querySelector('#ssr-container');
+    const containerDOMRect = containerElement.getBoundingClientRect();
+
+    return true;
+
+    // return (
+    //   containerDOMRect.left <= componentDOMRect.left &&
+    //   containerDOMRect.right >= componentDOMRect.right &&
+    //   containerDOMRect.top <= componentDOMRect.top &&
+    //   containerDOMRect.bottom >= componentDOMRect.bottom);
+  }
```

Comment:
Can you remind me why this is broken?

Reply:
Same reasoning as stated here: https://github.com/divmain/lightning-labs-alt/pull/1#discussion_r1533017060